presto-main/src/main/java/io/prestosql/server/security/KerberosConfig.java

```diff
@@ -99,4 +100,16 @@ public KerberosConfig setNameType(KerberosNameType nameType)
         this.nameType = nameType;
         return this;
     }
+
+    public String getUserExtraction()
```

Comment:
`Optional<String>`

Reply:
Last time I tried to use `Optional` is a config blew up.  I'll give it another shot